roles/ipaclient/library/ipaclient_test_keytab.py

```diff
@@ -159,11 +159,31 @@ def main():
     ca_crt_exists = os.path.exists(paths.IPA_CA_CRT)
     env = {'PATH': SECURE_PATH, 'KRB5CCNAME': paths.IPA_DNS_CCACHE}
 
-    # First try: Validate krb5 keytab with system krb5 configuraiton
+    # First try: Validate with temporary test krb5.conf that forces
+    # 1) no DNS lookups and
+    # 2) to load /etc/krb5.conf:
+    #
+    # [libdefaults]
+    # dns_lookup_realm = false
+    # dns_lookup_kdc = false
+    # include /etc/krb5.conf
+    #
     try:
+        (krb_fd, krb_name) = tempfile.mkstemp()
+        lines = [
+            "[libdefaults]",
+            "dns_lookup_realm = false",
+            "dns_lookup_kdc = false",
+            "include /etc/krb5.conf"
+        ]
+        with open(krb_name, "w") as outf:
+            for line in lines:
```

Comment:
Is it expected that `lines` be changed or be user configurable in the future?

If not, wouldn't it be better to use `"\n".join([.....])` and only `outf.write(line)`? Or does it break with Python 2.7?